### ISSUE TITLE
No predictions.csv created after running the sample experiments

### ISSUE THREAD

**Issue by shubham8garg** [alon-albalak/TLiDB#5]
I ran the sample baseline experiment and sample fine-tuning experiment both with run_experiment.py and tlidb command. I can see the folders get created with the best_model.pt and log.txt however, there is no predictions.csv created.
I made sure that the scripts had --do_eval flag set.

Am I missing anything?

**Comment by alon-albalak** [comment]
Yes, if you want to save the model predictions, you should also use the --save_pred flag

To see the exact scripts that will recreate our baselines, see https://github.com/alon-albalak/TLiDB/blob/master/tlidb/examples/FETA_sample_experiments.sh
